src/code42cli/cmds/auditlogs.py

```diff
@@ -0,0 +1,123 @@
+import click
+
+from code42cli.date_helper import parse_max_timestamp
+from code42cli.date_helper import parse_min_timestamp
+from code42cli.options import OrderedGroup
+from code42cli.options import sdk_options
+
+
+filter_option_usernames = click.option(
+    "--usernames", required=False, help="Filter results by usernames.", multiple=True,
```

Comment:
since we set `multiple=True`, it allows us to specify more than one usernames in format....
`--usernames "u1" --usernames "u2" ` in such case should the argument name be singular or plural?? e.g `--usernames` or `--username`

Reply:
I would stick with the same convention that we have on other commands. I believe everything else uses singular names, so we should be consistent and do that here, too.

Reply:
changed.